compiler-rt/lib/ctx_profile/CtxInstrContextNode.h

```diff
@@ -112,6 +112,12 @@ class ContextNode final {
 
   uint64_t entrycount() const { return counters()[0]; }
 };
+
+class ProfileWriter {
```

Comment:
Add a comment to describe the interface and expected usage?

Reply:
done - briefly. Updated further up the change stack, where more members appear.